### ISSUE TITLE
Memory leak for VolatileField Class, method finalize()

### ISSUE THREAD

**Issue by Tauchet** [dmulloy2/ProtocolLib#950]
**Describe the bug**
One way to explain the bug is that after some time, thanks to one of its used classes, the memory is left with residues of worlds that want to download, therefore in games that are widely downloading and loading preloaded worlds, it stays in memory and therefore over time, this leads to a bottleneck that could cause the server to crack or just shut down.

**To Reproduce**
Basically it occurs when worlds are unloaded in on-time, it causes the worlds to be trapped in containers of the VolatileField class for some reason and since the class is modified it is not finalized and therefore a reference remains in memory.

Basically researching a little about the code and seeing, that they make use of the finalize method, is there some way that the real method is executed, because without this worlds or any reference, they get trapped by that class, then it is a danger for the ram memory .

![image](https://user-images.githubusercontent.com/8809947/91106764-1e0e4f00-e639-11ea-8a54-22a20b83e745.png)

![image](https://user-images.githubusercontent.com/8809947/91107085-04213c00-e63a-11ea-86c9-47099efb8e51.png)

4.5.1

**Comment by OmerBasriKAYA** [comment]
i have same problem
![image](https://user-images.githubusercontent.com/29408163/103451532-142ae580-4cd6-11eb-8c5a-82385175df03.png)
![image](https://user-images.githubusercontent.com/29408163/103451543-3a508580-4cd6-11eb-92e6-93ee642714e8.png)
